pkg/version/root.go

```diff
@@ -68,3 +69,16 @@ func (c *RootCommand) Exec(in io.Reader, out io.Writer) error {
 	fmt.Fprintf(out, "Built with %s\n", GoVersion)
 	return nil
 }
+
+// PreRelease determines if the given app version is a pre-release.
+//
+// NOTE: this is indicated by the presence of a hyphen, e.g. v1.0.0-rc.1
+func PreRelease(version string) bool {
+	var result bool
+
+	if i := strings.Index(version, "-"); i >= 0 {
+		result = true
+	}
+
+	return result
```

Comment:
Is there a reason why you chose this over `strings.Contains()`?

```suggestion
    result strings.Contains(version, "-")
```

Reply:
no reason. should have used `Contains` so good shout 👍🏻